lint/unslice_checker.go

```diff
@@ -0,0 +1,38 @@
+package lint
+
+import (
+	"go/ast"
+	"go/types"
+)
+
+// unsliceChecker finds slice expressions that can be
+// simplified to sliced expression.
+func unsliceCheck(ctx *context) func(*ast.File) {
```

Comment:
We forgot to update function comment after function rename.
Comment says `unsliceChecker` when it should be `unsliceCheck`.